### PR TITLE
First layout

### PR DESCRIPTION
**Pull request by michaklr**
Start building backend
+
Adjustments on frontend

### CODE REVIEW

**Review by christian-porsch — COMMENTED** [review]
everything is fine

File: frontend/src/App.js [hunk 1]
```diff
@@ -8,6 +9,9 @@ function App() {
           <Route path={'/'}exact>
             <HomePage />
           </Route>
+            <Route path={'/Instructions'}exact>
```
Comment (fabianschmauder) [inline frontend/src/App.js]:
```suggestion
            <Route path={'/instructions'}exact>
```
i would use a small path name

File: frontend/src/pages/InstructionsPage.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default function InstructionsPage (){
+    return null
```
Comment (fabianschmauder) [inline frontend/src/pages/InstructionsPage.js]: i would show at least a tag with a title

**Review by fabianschmauder — COMMENTED** [review]
Looks good. I think you have a packe-lock.json on your root folder. Please remove it before merging

File: frontend/src/common/BulletList.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  h3 {
+    text-align: left;
+    margin: 0;
+  }
+
+  p {
+    margin: 12px 0 0 0;
+  }
```
Comment (ghost) [inline frontend/src/common/BulletList.js]: wird das so übernommen, auch wenn du oben im ListContainer kein li, h3 oder p hast? bin mir aber nicht sicher..

File: frontend/src/common/HomepageCard.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+HomepageCard.propTypes = {
+    children: PropTypes.any.isRequired,
+}
+
+export default function HomepageCard({ children }) {
+    return <CardBG>{children}</CardBG>
+}
+
+const CardBG = styled.div`
+  display: block;;
```
Comment (ghost) [inline frontend/src/common/HomepageCard.js]:
```suggestion
  display: block;
```
meckert er bei zwei ;; ?

File: frontend/src/components/Page.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components/macro'
+
```
Comment (ghost) [inline frontend/src/components/Page.js:2]: da kommt wsl noch inhalt, aber schonmal schön gestyled 👍

File: frontend/src/pages/ErgebnisPage.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+                        takimata sanctus est Lorem ipsum dolor sit amet.
+                    </p>
+                </CardContainer>
+            </Wrapper>
+            <button><Link to={'/manual'}>Start Test</Link></button>
+        </Page>
+    )
+}
+const CardContainer = styled.section`
+  display: block;;
```
Comment (ghost) [inline frontend/src/pages/ErgebnisPage.js]:
```suggestion
  display: block;
```
gleiche wie oben, doppelte ;;

File: frontend/src/pages/ManualPage.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+                        takimata sanctus est Lorem ipsum dolor sit amet.
+                    </p>
+                </CardContainer>
+            </Wrapper>
+            <button><Link to={'/manual'}>Start Test</Link></button>
+        </Page>
+    )
+}
+const CardContainer = styled.section`
+  display: block;;
```
Comment (ghost) [inline frontend/src/pages/ManualPage.js]:
```suggestion
  display: block;
```

File: frontend/src/pages/ManualPage.js [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+  position: relative;
+  background: white;
+  border-radius: 6px;
+  padding: 15px;
+  -webkit-box-shadow: 0px 0px 5px 4px rgba(0,0,0,0.1); 
+  box-shadow: 0px 0px 5px 4px rgba(0,0,0,0.1);
+`
+
+const Wrapper = styled.section`
+background-color: #CCE3DE
```
Comment (ghost) [inline frontend/src/pages/ManualPage.js:75]:
```suggestion
background-color: #CCE3DE;
```

**Review by ghost — APPROVED** [review]
Sieht schon nach richtig viel Arbeit aus, TOP!

File: frontend/package.json [hunk 9]
```diff
@@ -37,6 +38,9 @@
   },
   "proxy": "http://localhost:8080",
   "devDependencies": {
+    "@material-ui/core": "^4.11.4",
```
Comment (fabianschmauder) [inline frontend/package.json]: you should "@material-ui/core" and "react-router-dom" move to the normal dependencies. You also need them in production.

File: frontend/src/App.js [hunk 10]
```diff
@@ -8,6 +10,10 @@ function App() {
           <Route path={'/'}exact>
             <HomePage />
           </Route>
+            <Route path={'/manual'}exact>
+                <ManualPage />
+            </Route>
+            <ErgebnisPage />
```
Comment (fabianschmauder) [inline frontend/src/App.js]: you are missing a route here for the "ErgebnisPage" I would also rename it to some thing like "ResultPage"

File: frontend/src/common/BulletList.js (hunk 3 above)
Comment (fabianschmauder) [reply frontend/src/common/BulletList.js]: it also styles the children.  See Homepage ;)

File: frontend/src/pages/HomePage.js [hunk 11]
```diff
@@ -1,3 +1,145 @@
...
+            </p>
+            <Collapse headline="About Helping Hands">
+                <p>
+                    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+                </p>
+            </Collapse>
+
+<CardContainer>
```
Comment (fabianschmauder) [inline frontend/src/pages/HomePage.js]: you should use prettier to format you code

**Review by fabianschmauder — COMMENTED** [review]
LGTM 🎈

**Pull request merged**